lib/logstash/outputs/scalyr.rb

```diff
@@ -839,7 +839,7 @@ def build_multi_event_request_array(logstash_events)
       if @flatten_nested_values
         start_time = Time.now.to_f
         begin
-          record = Scalyr::Common::Util.flatten(record, delimiter=@flatten_nested_values_delimiter, flatten_arrays=@flatten_nested_arrays, fix_deep_flattening_delimiters=@fix_deep_flattening_delimiters, max_key_count=@flattening_max_key_count)
```

Comment:
This was invalid code - those function arguments are positional and not keyword arguments and that code was assigning local variables instead and not passing in keyword arguments.